node/src/blockchain/blockchain_interface.rs

```diff
@@ -2223,19 +2236,21 @@ mod tests {
             TEST_DEFAULT_CHAIN,
         );
 
-        match subject.get_block_number() {
+        let expected_error = match subject.get_block_number() {
```

Comment:
I think you can do `subject.get_block_number().unwrap_err()` here and ditch the `match`, if you want.